src/tiled/mainwindow.cpp

```diff
@@ -1328,6 +1329,7 @@ void MainWindow::updateRecentFilesMenu()
         mRecentFiles[i]->setText(QFileInfo(files[i]).fileName());
         mRecentFiles[i]->setData(files[i]);
         mRecentFiles[i]->setVisible(true);
+        mRecentFiles[i]->setToolTip(QFileInfo(files[i]).filePath());
```

Comment:
Just a small suggestion: please re-use the QFileInfo instance. Not that the code is performance-critical, but in general it's good to avoid duplicating work.

Reply:
Sure, I just updated